flixel/input/gamepad/id/MFiID.hx

```diff
@@ -25,8 +25,8 @@ class MFiID
 	public static inline var DPAD_LEFT:Int = 19;
 	public static inline var DPAD_RIGHT:Int = 20;
 	
-	public static var LEFT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(0, 1);
-	public static var RIGHT_ANALOG_STICK(default, null) = new FlxGamepadAnalogStick(2, 3);
+	public static var LEFT_ANALOG_STICK (default, null) = new FlxGamepadAnalogStick(0, 1, {up:21, down:22, left:23, right:24});
```

Comment:
Sure, that's totally fine, as long as you're okay with these other two considerations:

- The "FAKE" id's for legacy are adjacent, we'd be departing from that convention
- These ID's ultimately map to array indeces, so it'd implicitly create a lot of empty space in the underyling array structures